whats-new-in-pgstream-0-4-0.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'pgstream v0.4.0: Postgres-to-Postgres replication, snapshots & transformations'
+description: pgstream v0.4 includes Postgres to Postgres replication, snapshots and transformations.
```

Comment:
I think the title and the description say the same, thoughts on maybe changing the title to something simpler, like for pgroll updates?
`pgstream 0.4.0 update`
@CezzaineZ @alexfrancoeur